### ISSUE TITLE
CFRunLoopTimers Do Not Work in 'kCFRunLoopDefaultMode' in opencflite-635.21.5

### ISSUE THREAD

**Issue by gerickson** [gerickson/opencflite#56]
While the `CFRunLoopTimerExample` works in both opencflite-476 and opencflite-635, real, production applications that use `kCFRunLoopDefaultMode`, such as [hlxsimd](https://github.com/gerickson/openhlx/blob/main/src/hlxsimd/ApplicationController.cpp#L272), for their timer mode break in opencflite-635 whereas they work in opencflite-476.

**Comment by gerickson** [comment]
The failure is likely due to these bits of code in _CFRunLoop.c_:

```
@@ -2279,7 +2279,7 @@ static int32_t __CFRunLoopRun(CFRunLoopRef rl, CFRunLoopModeRef rlm, CFTimeInter
                 goto handle_msg;
             }
 #elif DEPLOYMENT_TARGET_LINUX || DEPLOYMENT_TARGET_FREEBSD
-            // Implement Me!
+#error "Implement me!"
 #endif
         }
 
@@ -2310,7 +2310,7 @@ static int32_t __CFRunLoopRun(CFRunLoopRef rl, CFRunLoopModeRef rlm, CFTimeInter
         // Note: don't pass 0 for polling, or this thread will never yield the CPU.
         __CFRunLoopWaitForMultipleObjects(waitSet, NULL, poll ? 0 : TIMEOUT_INFINITY, rlm->_msgQMask, &livePort, &windowsMessageReceived);
 #elif DEPLOYMENT_TARGET_LINUX || DEPLOYMENT_TARGET_FREEBSD
-        // XXX - More to fill in here.
+#error "Implement me!"
 #endif
         
    __CFRunLoopLock(rl);
@@ -2356,7 +2356,7 @@ static int32_t __CFRunLoopRun(CFRunLoopRef rl, CFRunLoopModeRef rlm, CFTimeInter
        sourceHandledThisLoop = true;
         } else
 #elif DEPLOYMENT_TARGET_LINUX
-            // XXX - More to fill in here.
+#error "Implement me!"
 #endif
    if (MACH_PORT_NULL == livePort) {
             // handle nothing
```

**Comment by gerickson** [comment]
Closed with #86.
